{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/layout.jinja2

```diff
@@ -11,15 +11,15 @@
     <title>Cookiecutter Starter project for the Pyramid Web Framework</title>
 
     <!-- Bootstrap core CSS -->
-    <link href="//oss.maxcdn.com/libs/twitter-bootstrap/3.0.3/css/bootstrap.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
 
     <!-- Custom styles for this scaffold -->
     <link href="{{ '{{request.static_url(' }}'{{cookiecutter.repo_name}}:static/theme.css'){{ '}}' }}" rel="stylesheet">
 
     <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
     <!--[if lt IE 9]>
       <script src="//oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js" integrity="sha384-0s5Pv64cNZJieYFkXYOTId2HMA2Lfb6q2nAcx2n0RTLUnCAoTTsS0nKEO27XyKcY" crossorigin="anonymous"></script>
-      <script src="//oss.maxcdn.com/libs/respond.js/1.3.0/respond.min.js" integrity="sha384-f1r2UzjsxZ9T4V1f2zBO/evUqSEOpeaUUZcMTz1Up63bl4ruYnFYeM+BxI4NhyI0" crossorigin="anonymous"></script>
+      <script src="//oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
```

Comment:
It looks like all the respond.js links got integrity removed.. intentional?

Reply:
@mmerickel  No, I just can't find it on oss.maxcdn.com for some reason. I'd be happy to add it back if you know what the SHA is.

Reply:
Can't you just download the file and pipe it through `shasum -a 384` ? Maybe I'm missing something.

Reply:
`66868c6c317ee0b785c60e96752710f679d1d500b63084710353bd2961174309ecd46f143722046484227736f44f57e8`

Reply:
Sorry it's `sha384-ZoaMbDF+4LeFxg6WdScQ9nnR1QC2MIRxA1O9KWEXQwns1G8UNyIEZIQidzb0T1fo`.

Reply:
Confirmed using this tool from Mozilla, which also has the command to use:
https://www.srihash.org/

`openssl dgst -sha384 -binary FILENAME.js | openssl base64 -A`